packages/framework/src/parser.ts

```diff
@@ -114,7 +115,14 @@ export const EnsembleParser = {
     const viewWidget = unwrapWidget(viewNode);
     const apis = unwrapApiModels(screen);
 
-    const global = get(screen, "Global");
+    const globalBlock = get(screen, "Global");
+    const scriptName = get(globalBlock, "scriptName");
+    let global: string | undefined = globalBlock;
+    if (scriptName) {
+      global = app.scripts?.find((script) => script.name === scriptName)
+        ?.content;
+    }
```

Comment:
Hey, so here is all needed validation is added. If global block is not contain the scriptName, then it will be use a string from the global scope and even if there will be no script file existence while passing the scriptName, then it will become undefined

Reply:
What if global block is not a string? What if scriptName is not a string?